src/main/steps/tasklistresponse/miam/MIAMPostController.ts

```diff
@@ -0,0 +1,46 @@
+import autobind from 'autobind-decorator';
+import type { Response } from 'express';
+
+//import { getSystemUser } from '../../../app/auth/user/oidc';
+import { CosApiClient } from '../../../app/case/CosApiClient';
+import { Respondent } from '../../../app/case/definition';
+import { toApiFormat } from '../../../app/case/to-api-format';
+import type { AppRequest } from '../../../app/controller/AppRequest';
+import { AnyObject, PostController } from '../../../app/controller/PostController';
+import { FormFields, FormFieldsFn } from '../../../app/form/Form';
+import { RESPONDENT_TASK_LIST_URL } from '../../urls';
+
+import { setMIAMDetails } from './MIAMMapper';
+
+@autobind
+export class MIAMPostController extends PostController<AnyObject> {
+  constructor(protected readonly fields: FormFields | FormFieldsFn) {
+    super(fields);
+  }
+  public async post(req: AppRequest, res: Response): Promise<void> {
+    const caseworkerUser = req.session.user;
```

Comment:
should the variable name be 'loggedInCitizen'?

Reply:
done